Doc/comment server code

albertinisg opened the pull request:
This PR fixes #85

Commit:
Add comments for the utils.js functions

Commit:
Remove unused function from utils.js

Commit:
Add jsdoc comments in every function of Tourguide server

dmabtrg committed:
Add comments for the functions.

dmabtrg committed:
Small refactoring of the sensors feeders.

dmabtrg committed:
Remove commented code.

dmabtrg committed:
Use console.error instead of console.log for error messages.

FiwareULPGC commented on server/auth/auth.js:
```diff
@@ -31,8 +31,12 @@ var oauth = new OAuth2(clientId,
+*/
 exports.login = function(req, res) {
   // Using the access code goes again to the IDM to obtain the accessToken
   oauth.getOAuthAccessToken(req.query.code, function(e, results) {
```
(Line 43) Unneeded line

FiwareULPGC commented on server/auth/auth.js:
```diff
@@ -53,13 +57,23 @@ exports.login = function(req, res) {
 
-// Redirect the user to the IdM for authentication
+/**
+ * Redirect the user to the IdM for Oauth authentication
```
nit: OAuth

FiwareULPGC commented on server/auth/auth.js:
```diff
@@ -113,6 +140,12 @@ exports.validateRequest = function(req, res, next) {
+ *
+ * @param {Object} req - Request received
+ * @return {Promise} returns the user data using promises
+*/
```
We don't fully understand this comment

albertinisg replied:
This is the same as the function `getUserData`, but instead of having a response (`res`), we return a Promise using the [Q library](https://github.com/kriskowal/q) for a better handling of the user information

FiwareULPGC commented on server/auth/authrequest.js:
```diff
@@ -21,6 +21,16 @@ var port = 1026;
+ * @return {Promise} promise with the result of the request
+*/
 function performRequest(endpoint, method, data, fiwareHeaders, querystring) {
 
```
Unneeded line

FiwareULPGC commented on server/auth/authrequest.js:
```diff
@@ -21,6 +21,16 @@ var port = 1026;
+*/
 function performRequest(endpoint, method, data, fiwareHeaders, querystring) {
 
   var deferred = Q.defer();
```
(Lines 39-43) Maybe is better to use:

```
if (fiwareHeaders) {
  if (fiwareHeaders['fiware-service']) {
    .....
  }
  if (fiwareHeaders['fiware-servicepath']) {
    .....
  }
}

```

(Lines 60-77) nit: indentation

FiwareULPGC commented on server/feeders/occupancyupdater.js:
```diff
@@ -132,7 +132,7 @@ function getRestaurantReservationsByDate(restaurant, from, to) {
+          console.error('ERROR: Failed to get reservations:', err.message);
           return null;
         }
       );
```
(Line 173) Unneeded line

FiwareULPGC commented on server/feeders/occupancyupdater.js:
```diff
@@ -132,7 +132,7 @@ function getRestaurantReservationsByDate(restaurant, from, to) {
         })
```
(Lines 115-166) Please use `/**/` for multiline comments

FiwareULPGC commented on server/feeders/occupancyupdater.js:
```diff
@@ -227,7 +227,7 @@ function updateRestaurants() {
+      console.error(err);
     } else {
       console.log('Done.');
     }
```
(Line 243) Unneeded line

FiwareULPGC commented on server/feeders/reservationsgenerator.js:
```diff
@@ -46,7 +46,7 @@ var feedOrionReservations = function() {
     utils.sendRequest('POST', attributes, null, fiwareHeaders)
```
(Lines 8-12) Need full stop at the end of line
(Line 38) Commented code

FiwareULPGC commented on server/feeders/reservationsgenerator.js:
```diff
@@ -46,7 +46,7 @@ var feedOrionReservations = function() {
+      console.error(err);
     });
   }, apiRestSimtasks);
 
```
(Line 58) Unneeded space
(Line 98) Unneeded space

FiwareULPGC commented on server/feeders/restaurantfeeder.js:
```diff
@@ -85,15 +85,15 @@ var feedOrionRestaurants = function() {
           utils.sendRequest('POST', attributes, null, fwHeaders)
```
(Lines 8-12) Need full stop at the end of line
(Line 29) nit number --> Number
(Line 61) Commented code

albertinisg replied:
There is an upcoming PR for the feeders to adapt it to the CLI, so I'll address all the comments for the feeders in another PR

FiwareULPGC commented on server/feeders/restaurantfeeder.js:
```diff
@@ -85,15 +85,15 @@ var feedOrionRestaurants = function() {
+            console.error(err);
           });
         });
        }, geoWaitTimeMs);
```
(Line 125) Unneeded line
(Line 171) Unneeded line
(Line 235) Unneeded line

FiwareULPGC commented on server/feeders/sensorsgenerator.js:
```diff
@@ -22,23 +22,26 @@ var fiwareHeaders = {
+ * @param {Array} restaurantsData - list of restaurants to process.
+*/
+function feedIDASSensors(restaurantsData) {
 
```
Unneeded line

FiwareULPGC commented on server/feeders/reviewsgenerator.js:
```diff
@@ -53,7 +53,7 @@ var feedOrionReviews = function() {
     utils.sendRequest('POST', attributes, null, fiwareHeaders)
```
(Lines 24-28) nit Capitalize
(Line 45) Commented code .

FiwareULPGC commented on server/feeders/reviewsgenerator.js:
```diff
@@ -53,7 +53,7 @@ var feedOrionReviews = function() {
+      console.error(err);
     });
   }, apiRestSimtasks);
 
```
(Line 66) Unneeded line
(Line 105) Unneeded line

FiwareULPGC commented on server/feeders/reviewsgenerator.js:
```diff
@@ -143,7 +143,7 @@ var triggerRestaurantsRatings = function() {
+      console.error(err.error);
     });
   });
 };
```
(Line 153) Unneeded line

FiwareULPGC commented on server/feeders/sensorsupdater.js:
```diff
@@ -22,28 +22,31 @@ var fiwareHeaders = {
+/**
+ * Update the sensors data for each restaurant.
+ *
+ * @param {Array} restaurantsData - list of restaurants to process.
```
nit list --> List

FiwareULPGC commented on server/feeders/sensorsupdater.js:
```diff
@@ -22,28 +22,31 @@ var fiwareHeaders = {
+ * @param {Array} restaurantsData - list of restaurants to process.
+*/
+function feedIDASSensors(restaurantsData) {
 
```
Unneeded line

FiwareULPGC commented on server/feeders/sensorsupdater.js:
```diff
@@ -66,8 +69,15 @@ var feedIDASSensors = function() {
+ *
+ * @param {Number} oldValue - old value of the sensor.
+ * @param {String} type - sensor type
+ * @return {Number} newValue - new sensor value
```
(Lines 77-79) nit Capitalize

FiwareULPGC commented on server/idas/ul20.js:
```diff
@@ -128,6 +133,14 @@ function createService() {
+ * @param {String} room - the room where the sensor will be locaed.
+ * @param {String} type - the type of the new sensor.
+ * @return {Promise} response of the IoT Agent.
+*/
```
(Lines 139-142)  Capitalize

FiwareULPGC commented on server/idas/ul20.js:
```diff
@@ -118,7 +123,7 @@ function createService() {
+    console.error('Error on request: ' + error.message);
     q.reject(error);
   });
 
```
(Line 130) nit perform --> Perform

FiwareULPGC commented on server/idas/ul20.js:
```diff
@@ -128,6 +133,14 @@ function createService() {
+*/
 function registerSensor(restaurant, room, type) {
   var idasUrl = '/iot/devices';
   var headers = JSON.parse(JSON.stringify(defaultHeaders));
```
(Line 149) nit build -> Build
(Lines 192, 194-196) Consider use console.error

FiwareULPGC commented on server/idas/ul20.js:
```diff
@@ -235,7 +253,6 @@ function sendObservation(deviceId, data, servicePath) {
-        //console.log('Measurement sent: ' + entityName + ' ' + data);
         q.resolve(responseString);
       } else {
         console.log('Request:', options);
```
(Lines 258-261) Consider use console.error

FiwareULPGC commented on server/idas/ul20.js:
```diff
@@ -197,6 +210,14 @@ function registerSensor(restaurant, room, type) {
+ *
+ * @param {String} deviceId - Id of the sensor in the IoT Agent.
+ * @param {String} data - the measurement data.
+ * @param {String} servicePath - the servicePath of the sensor.
```
(Lines 217-218) Capitalize

FiwareULPGC commented on server/idas/ul20.js:
```diff
@@ -258,20 +275,44 @@ function sendObservation(deviceId, data, servicePath) {
+ *
+ * @param {String} deviceId - Id of the sensor in the IoT Agent.
+ * @param {String} value - the temperature value.
+ * @param {String} servicePath - the servicePath of the sensor.
```
(Lines 282,283) Capitalize

FiwareULPGC commented on server/idas/ul20.js:
```diff
@@ -258,20 +275,44 @@ function sendObservation(deviceId, data, servicePath) {
+ *
+ * @param {String} deviceId - Id of the sensor in the IoT Agent.
+ * @param {String} value - the relative humidity value.
+ * @param {String} servicePath - the servicePath of the sensor.
```
(Lines 296-298) Capitalize

FiwareULPGC commented on server/idas/ul20.js:
```diff
@@ -258,20 +275,44 @@ function sendObservation(deviceId, data, servicePath) {
+ *
+ * @param {Object} restaurant - restaurant entity.
+ * @param {String} room - the room where the sensor will be locaed.
+ * @param {String} type - the type of the new sensor.
```
(Lines 311-313) Capitalize

FiwareULPGC commented on server/idas/ul20.js:
```diff
@@ -286,6 +327,15 @@ function initializeSensor(restaurant, room, type) {
+ * @param {String} deviceId - Id of the sensor in the IoT Agent.
+ * @param {String} type - the sensor type.
+ * @param {String} value - the measurement value.
+ * @param {String} servicePath - the servicePath of the sensor.
```
(Lines 333-336) Capitalize

FiwareULPGC commented on server/idas/ul20.js:
```diff
@@ -297,6 +347,14 @@ function updateSensor(deviceId, type, value, servicePath) {
+ *
+ * @param {Object} restaurant - restaurant entity of the sensor.
+ * @param {String} room - the room where the sensor will be locaed.
+ * @param {String} type - the type of the new sensor.
```
(Lines 353-355) Capitalize

FiwareULPGC commented on server/utils.js:
```diff
@@ -31,6 +31,14 @@ var FOOD_ESTABLISHMENT_TYPE = 'FoodEstablishment';
 var DATE_TYPE = 'Date';
```
(Line 7) Full stop at end of line

FiwareULPGC commented on server/utils.js:
```diff
@@ -31,6 +31,14 @@ var FOOD_ESTABLISHMENT_TYPE = 'FoodEstablishment';
+*/
 function doGet(options, callback, res, useHttps) {
   var protocol = http;
   if (useHttps) {
```
(Lines 49-50) Please use `/* */` for multiline comments.
(Line 49) nit data --> Data

albertinisg replied:
Function `doGet` was used in the old feeders, will be removed in an upcoming PR!

FiwareULPGC commented on server/utils.js:
```diff
@@ -54,7 +62,7 @@ function doGet(options, callback, res, useHttps) {
         // console.log(data);
```
(Line 62) Commented code

FiwareULPGC commented on server/utils.js:
```diff
@@ -54,7 +62,7 @@ function doGet(options, callback, res, useHttps) {
         // console.log(data);
       } catch (err) {
         console.log('Can\'t decode JSON response.');
```
(Line 64) Maybe console.log --> console.error

FiwareULPGC commented on server/utils.js:
```diff
@@ -54,7 +62,7 @@ function doGet(options, callback, res, useHttps) {
+        console.error(err);
         msg = 'Can\'t decode JSON response.';
       }
       if (data === undefined) {
```
(Line 68) Maybe is better to use `if (!data) {`
(Line 77) Maybe console.log --> console.error

FiwareULPGC commented on server/utils.js:
```diff
@@ -70,54 +78,18 @@ function doGet(options, callback, res, useHttps) {
+*/
 function replaceOnceUsingDictionary(dictionary, content,
   replacehandler) {
   if (typeof replacehandler !== 'function') {
```
(Lines 118-119) Please use `/* */`for multiline comments
(Lines 125,126) Where is the test?, also use `/**/`for multiline comments
(Line 126) nit: `we`--> `we` (there are two blank spaces)
(Line 145) Maybe is better a comment like `// In order to avoid a infinite loop, lastIndex is updated`

FiwareULPGC commented on server/utils.js:
```diff
@@ -195,6 +186,13 @@ function fixedEncodeURIComponent(str) {
+ * Returns a random date between given dates
+ *
+ * @param {String} from - datetime from
+ * @param {String} to - datetime to
```
(Lines 192-193) Capitalize

FiwareULPGC commented on server/utils.js:
```diff
@@ -209,6 +207,12 @@ function getRandomDate(from, to) {
 }
 
+/**
+ * Removes html code tags
```
nit HTML

FiwareULPGC commented on server/utils.js:
```diff
@@ -209,6 +207,12 @@ function getRandomDate(from, to) {
+ * Removes html code tags
+ *
+ * @param {String} str - String to clean
+ * @return {String} str - String without the html tags
```
nit HTML

FiwareULPGC commented on server/utils.js:
```diff
@@ -209,6 +207,12 @@ function getRandomDate(from, to) {
+*/
 function convertHtmlToText(str) {
 
   //-- remove BR tags and replace them with line break
```
Why do you have `--` here? It occurs several times in another comments lines 232,235,238,241....

FiwareULPGC commented on server/utils.js:
```diff
@@ -238,6 +242,12 @@ function convertHtmlToText(str) {
+ * @return {Object} element - Array with the element
+*/
 function objectToArray(element) {
 
```
Unneeded line

FiwareULPGC commented on server/utils.js:
```diff
@@ -238,6 +242,12 @@ function convertHtmlToText(str) {
+*/
 function objectToArray(element) {
 
   if (util.isArray(element) === false) {
```
Maybe is better to use something like `if (!utils.isArray(element)) {`

FiwareULPGC commented on server/utils.js:
```diff
@@ -249,6 +259,12 @@ function objectToArray(element) {
+ * Adds the measurements units to the sensor's schema
+ *
+ * @param {Object} sensor - sensor to add the units to
+ * @return {Object} sensor - sensor with the units added
```
(Lines 265-266) Capitalize

FiwareULPGC commented on server/utils.js:
```diff
@@ -265,6 +281,12 @@ function setSchemaUnits(sensor) {
+ * @return {Object} elementToSchema - Object in schema.org format
+*/
 function restaurantToSchema(element, date) {
 
```
Uneeded space

FiwareULPGC commented on server/utils.js:
```diff
@@ -265,6 +281,12 @@ function setSchemaUnits(sensor) {
+*/
 function restaurantToSchema(element, date) {
 
   var restaurantSchemaElements = [
```
(Line 339) Unneeded line

FiwareULPGC commented on server/utils.js:
```diff
@@ -357,6 +379,12 @@ function restaurantToSchema(element, date) {
+ * @return {Object} elementToSchema - Object in schema.org format
+*/
 function reviewToSchema(element) {
 
```
Unneeded line

FiwareULPGC commented on server/utils.js:
```diff
@@ -386,6 +414,12 @@ function reviewToSchema(element) {
+ * @return {Object} elementToSchema - Object in schema.org format
+*/
 function reservationToSchema(element) {
 
```
Unneeded line

FiwareULPGC commented on server/utils.js:
```diff
@@ -416,6 +450,13 @@ function reservationToSchema(element) {
 
```
(Lines 448, 450) Unneeded lines

FiwareULPGC commented on server/utils.js:
```diff
@@ -416,6 +450,13 @@ function reservationToSchema(element) {
+ * Convert elements with Orion format into schema.org ones
+ *
+ * @param {Object} element - Object to convert
+ * @param {String} date - datetime
```
Capitalize

FiwareULPGC commented on server/utils.js:
```diff
@@ -416,6 +450,13 @@ function reservationToSchema(element) {
+ * @return {Object} newElement - Object in schema.org format
+*/
 function objectDataToSchema(element, date) {
 
```
Unneeded line

FiwareULPGC commented on server/utils.js:
```diff
@@ -522,6 +577,14 @@ function completeAddress(schemaObject, geoObject) {
+ * Completes the schemaObject location element
+ *
+ * @param {Object} schemaObject - Object to be added in Orion
+ * @param {Object} geoObject - geocoder object
```
Capitalize

FiwareULPGC commented on server/utils.js:
```diff
@@ -665,6 +752,13 @@ function reservationToOrion(userObject, schemaObject) {
+ * Generate a list of reservations of a Franchise
+ *
+ * @param {List} listOfRestaurants - restaurants from a Franchise
+ * @param {List} listOfReservations - list of reservations
```
(Lines 758-759) Capitalize

FiwareULPGC replied:
It still needs to be capitalized.

FiwareULPGC commented on server/utils.js:
```diff
@@ -693,6 +797,16 @@ function getListByType(type, element, headers, queryString, normalized) {
+ *
+ * @param {String} method - method to use
+ * @param {Object} body - body to send
+ * @param {String} identifier - identifier of the entity
```
(Lines 803-805) Capitalize

FiwareULPGC replied:
It still needs to be capitalized.

FiwareULPGC commented on server/utils.js:
```diff
@@ -741,6 +867,12 @@ function getAggregateRating(listOfReviews) {
+ * Generates a String with the sql sentence for date filtering
+ *
+ * @param {String} isoTimeString - datetime
+ * @return {String} frameTime - string to filter
```
(Lines 873-874) Capitalize

FiwareULPGC commented on server/utils.js:
```diff
@@ -749,13 +881,26 @@ function getTimeframe(isoTimeString) {
+ * Generates a String with the sql sentence for date filtering
+ *
+ * @param {String} from - datetime from
+ * @param {String} to - datetime to
```
(Lines 887-889) Capitalize

FiwareULPGC commented on server/utils.js:
```diff
@@ -749,13 +881,26 @@ function getTimeframe(isoTimeString) {
+ * Calculates the sum of the occupancyLevels
+ *
+ * @param {List} listOfReservations - reservations list
+ * @return {Integer} occupancyLevels - the occupancyLevels
```
(Lines 901-902) Capitalize

FiwareULPGC commented on server/utils.js:
```diff
@@ -767,6 +912,13 @@ function getOccupancyLevels(listOfReservations) {
+ *
+ * @param {Integer} occupancyLevel - The occupancyLevels
+ * @param {Date} date - date object
+ * @return {Object} occupancyObject - object to update a Restaurant
```
(Lines 919-920) Capitalize

FiwareULPGC commented on server/utils.js:
```diff
@@ -783,12 +935,26 @@ function createOccupancyObject(occupancyLevel, date) {
+ *
+ * @param {List} listOfReservations - Reservations list
+ * @param {Date} date - date object
+ * @return {Object} occupancyLevelsObj - object to update a Restaurant
```
(Lines 942-943) Capitalize

FiwareULPGC commented on server/utils.js:
```diff
@@ -783,12 +935,26 @@ function createOccupancyObject(occupancyLevel, date) {
+ *
+ * @param {String} name - element name
+ * @param {Date} date - date object
+ * @return {String} id - identifier based in the parameters
```
(Lines 954-956) Capitalize

FiwareULPGC commented on server/utils.js:
```diff
@@ -803,6 +969,15 @@ function generateId(name, date) {
+ * @param {String} key - key to filter
+ * @param {String} operator - operator for the matching
+ * @param {String} value - value
+ * @return {List} listOfConditions - updated list of conditions
```
(Lines 975-979) Capitalize

FiwareULPGC commented on server/utils.js:
```diff
@@ -820,6 +995,13 @@ function addConditionToQuery(listOfConditions, key, operator, value) {
+ *
+ * @param {Object} headers - headers of the request
+ * @param {String} department - fiware-servicepath (FranchiseX)
+ * @return {Object} fiwareHeaders - updated headers
```
(Lines 1001-1003) Capitalize

FiwareULPGC commented on server/utils.js:
```diff
@@ -828,6 +1010,12 @@ function completeHeaders(headers, department) {
+ * Removes the fiware-servicepath for reviews and restaurants
+ *
+ * @param {Object} headers - headers of the request
+ * @return {Object} fiwareHeaders - updated headers
```
(Lines 1006-1007) Capitalize

FiwareULPGC commented on server/utils.js:
```diff
@@ -836,6 +1024,12 @@ function removeServicePath(headers) {
+ * Send the response object in schema.org format
+ *
+ * @param {Object} data - object from Orion
+ * @param {Object} res - response to send
```
(Lines 1030-1031) Capitalize

FiwareULPGC commented on server/utils.js:
```diff
@@ -846,6 +1040,12 @@ function returnResponse(data, res, date) {
+ * Send the response error
+ *
+ * @param {Object} err - error object
+ * @param {Object} res - response to send
```
(Lines 1046-1047) Capitalize

FiwareULPGC commented on server/utils.js:
```diff
@@ -858,6 +1058,13 @@ function responseError(err, res) {
+ *
+ * @param {Object} data - data object
+ * @param {String} element - element added
+ * @param {Object} res - response to send
```
(Lines 1064-1066) Capitalize

FiwareULPGC commented on server/utils.js:
```diff
@@ -871,6 +1078,12 @@ function responsePost(data, element, res) {
+ * Send error response when the schema is invalid
+ *
+ * @param {Object} res - response to send
+ * @param {Object} tv4 - schema validator object
```
(Lines 1084-1085) Capitalize

FiwareULPGC commented on server/utils.js:
```diff
@@ -884,6 +1097,11 @@ function returnInvalidSchema(res, tv4) {
+/**
+ * Send error response when the resource is forbidden
+ *
+ * @param {Object} res - response to send
```
Capitalize

FiwareULPGC commented on server/utils.js:
```diff
@@ -895,6 +1113,11 @@ function returnForbidden(res) {
+/**
+ * Send error response when there's a conflict
+ *
+ * @param {Object} res - response to send
```
Capitalize

FiwareULPGC commented on server/utils.js:
```diff
@@ -532,6 +595,14 @@ function addGeolocation(schemaObject, geoObject) {
+ *         converted to Context Broker data model
+*/
 function restaurantToOrion(schemaObject, geoObject) {
 
```
Unneeded line

FiwareULPGC commented on server/utils.js:
```diff
@@ -593,6 +664,14 @@ function restaurantToOrion(schemaObject, geoObject) {
+ *         converted to Context Broker data model
+*/
 function reviewToOrion(userObject, schemaObject) {
 
```
Unneeded line

FiwareULPGC commented on server/utils.js:
```diff
@@ -629,6 +708,14 @@ function reviewToOrion(userObject, schemaObject) {
+ *         converted to Context Broker data model
+*/
 function reservationToOrion(userObject, schemaObject) {
 
```
Unneeded line

FiwareULPGC commented on server/utils.js:
```diff
@@ -665,6 +752,13 @@ function reservationToOrion(userObject, schemaObject) {
+ * @return {List} contains all the reservations of the passed restaurants
+*/
 function getOrgReservations(listOfRestaurants, listOfReservations) {
 
```
Unneeded line

FiwareULPGC commented on server/utils.js:
```diff
@@ -713,6 +833,12 @@ function getAverage(data) {
+ * @return {Object} newElement - The average element updated
+*/
 function getAggregateRating(listOfReviews) {
 
```
Unneeded line

FiwareULPGC commented on server/utils.js:
```diff
@@ -749,13 +881,26 @@ function getTimeframe(isoTimeString) {
+ * @return {Integer} occupancyLevels - the occupancyLevels
+*/
 function getOccupancyLevels(listOfReservations) {
 
```
Unneeded line

FiwareULPGC commented:
LGTM except for the mentioned details:

- Unneeded new lines
- Indentation
- Usage of capital letter at the beginning of a comment

Thanks

Commit:
Several bugfixes in comments, empty lines and capital letters

albertinisg commented:
Do not merge yet, there's an upcoming commit from @dmabtrg

dmabtrg committed:
Remove unneeded script occupancyupdater.js.

The functionality provided by this script is now done by the application
on demand.

dmabtrg committed:
Fix comments for sensors feeders.

dmabtrg committed:
Use console.error for error messages.

albertinisg commented:
@FiwareULPGC changes done and comments addressed!

FiwareULPGC commented on server/idas/ul20.js:
```diff
@@ -175,19 +188,19 @@ function registerSensor(restaurant, room, type) {
+    console.error('Error on request: ' + error.message);
     q.reject(error);
   });
 
```
Line 207 - nit Perform

FiwareULPGC commented on server/idas/ul20.js:
```diff
@@ -235,20 +253,19 @@ function sendObservation(deviceId, data, servicePath) {
+    console.error('Error on request: ' + error.message);
     q.reject(error);
   });
 
```
Line 272 - nit Perform

FiwareULPGC commented on server/utils.js:
```diff
@@ -411,13 +434,17 @@ function reservationToSchema(element) {
-
   var newElement;
   var type = element.type;
 
```
lines 451-474 Indentation

Commit:
Minor indentation and capital letters bugfixes

albertinisg commented:
@FiwareULPGC comments reviewed!

FiwareULPGC commented:
LGTM
